models/Student.js

```diff
@@ -12,7 +12,7 @@ const StudentSchema = new Schema({
     required: true
   },
   grade: {
-    type: Number,
+    type: String,
```

Comment:
This is good for now. In the future, we will probably want to make this into an enumerated type (or enumeration).

- [More info about enumerations in general](https://en.wikipedia.org/wiki/Enumerated_type)
- [How to make `enum` values in Mongoose](https://thewebdev.info/2022/02/28/how-to-create-and-use-enums-in-mongoose/)

If we had an enumeration, we could automatically generate the options in the select list instead of only storing them on the template.